.github/workflows/checks.yml

```diff
@@ -49,5 +47,8 @@ jobs:
         run: yes | poetry run mypy cashu --install-types || true
       - name: Run mypy
         run: poetry run mypy cashu --ignore-missing
-      - name: Run flake8
-        run: poetry run flake8
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: chartboost/ruff-action@v1
```

Comment:
Neat that the `ruff-action` 'just works', but this isn't necessarily going to be the same version as we'd be using locally with poetry. Also I assume this respects any ruff config in `pyproject.toml` but not sure?

Reply:
yep, it has to respect it, i tried it and else our isort and stuff like that wouldnt validate because its not default. but let see i dont have a lot of experience with it yet.